config/zend-expressive/zend-expressive.config.php

```diff
@@ -0,0 +1,135 @@
+<?php
```

Comment:
Actually this config should be agnostic and not tied to zend expressive. You should simply call it "config.global.php". I've already seen bridge for Slim container that consume service manager container, so theorically this could be re-used for any other frameowkr, not just expressive